src/iiif-collection.json

```diff
@@ -962,7 +980,7 @@
           "@id": "https://iiif-commons.github.io/iiif-av-component/examples/data/iiif/lunchroom-manners.json",
           "@type": "sc:Manifest",
           "label": "Lunchroom manners",
-          "visible": true
+          "visible": false
```

Comment:
Are you intentionally hiding all of these manifests? If so, just curious why.

Reply:
Yes, but can delete them. I've just found in the past that after deleting something I then need it again and have to trawl through git history to find it.